cli/command/container/logs.go

```diff
@@ -3,9 +3,9 @@ package container
 import (
 	"io"
 
-	"github.com/docker/cli/cli"
-	"github.com/docker/cli/cli/command"
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/cli"
+	"github.com/docker/docker/cli/command"
```

Comment:
Yeah, I've just changed them back. I'll re-push what I have but the tests will stay broken.